test/sql/storage/catalog/test_storage_sequences.cpp

```diff
@@ -19,8 +19,12 @@ TEST_CASE("Use sequences over different runs without checkpointing", "[storage]"
 		REQUIRE_NO_FAIL(con.Query("CREATE SEQUENCE seq_cycle INCREMENT 1 MAXVALUE 3 START 2 CYCLE;"));
```

Comment:
I would like to see a lot more tests, see e.g. `test/sql/catalog/sequence/test_sequence.test` for the tests involving `nextval`. I would like to see tests involving different parameters for the function (e.g. `select currval(null)`, `select currval(i) from tbl`), and also how this works with different sequence types (e.g. cyclic sequences, sequences with positive and negative increments, sequences with limits).

Error cases also always need to be triggered in tests (so we need a test that triggers all exception indicated above, tests with a sequence that doesn't exist, etc).

In general we also prefer to use sqllogictests over the C++ tests.

Reply:
Ok, will do

Reply:
What is the easiest way to run the tests using cmake on windows and VS2019? As far as I can tell there is no way for me to get Windows to recognize the command `make unit`.

Reply:
Check the [CI config](https://github.com/duckdb/duckdb/blob/master/.github/workflows/main.yml#L100) for what we use on Windows. Note that you might need to add certain things (e.g. cmake) to your path for this to work.

Reply:
An alternative (if you prefer to use the GUI) is to use the cmake gui tool to generate a visual studio 2019 project. Then you can open the project in Visual Studio and build the "unittest" target.